tmt/steps/provision/testcloud.py

```diff
@@ -655,6 +666,16 @@ def guest(self) -> Optional[tmt.Guest]:
         """ Return the provisioned guest """
         return self._guest
 
+    def _print_local_images(self) -> None:
+        """ Print images which are already cached """
+        self.info("Locally available images")
+        try:
+            for f in sorted(os.listdir(TESTCLOUD_IMAGES)):
+                if f.endswith('.qcow2'):
+                    self.print(f"{TESTCLOUD_IMAGES / f}", shift=1)
```

Comment:
I’d advise against `info` and `print`, both are designed for *logging*, not output. The list of images would end up in `stderr` rather than in `stdout`. Use `click.echo` for now, I plan to provide a helper for command output later.

Reply:
Fixed in https://github.com/teemtee/tmt/pull/1997/commits/2f98babbca0d9ed3629f92e4f67e6794e2d9e7e6.